src/databases/MFEM/JSONRoot.C

```diff
@@ -899,20 +912,24 @@ JSONRoot::ToJson(ostringstream &oss)
     // loop over data sets
     for(int i=0;i<(int)dset_names.size();i++)
     {
+        std::string mesh_path = StringHelpers::EscapeSpecialChars(dset.Mesh().Path().Expand());
```

Comment:
'dset' is undeclared here, line 915 should be moved to below line 919 (or line 919 moved above this one.